digitalocean/hacktoberfest
# replace roles multi-select with checkboxes

# Description

i turned the multiple select field on the registration screen into checkboxes since it's more user friendly than a standard html multiple select. it's not clear to a normal user that you're supposed to ctrl+click in order to select multiple options. it was also letting users pick "Select all the roles that apply to you" as an option.
## old
![image](https://user-images.githubusercontent.com/57028336/95645784-e3d41080-0b05-11eb-8389-e43fad8513d0.png)
## new
![image](https://user-images.githubusercontent.com/57028336/95645827-33b2d780-0b06-11eb-9de0-b2217004880f.png)

# Test process


1. sign up and select checkboxes for your selected role
2. go to profile and click edit
3. ensure the selected checkboxes are the same ones signed up with

# Requirements to merge
